File: CCp/Arrays/Java/onestar/b/SmallFact.java
import java.util.Scanner;

/**
 * SmallFact
 */
public class SmallFact {

  public static void factorial(int n) {

    int fact = 1;
    for (int i = n; i >= 1; i--) {
      fact *= i;
    }
    System.out.println(fact);

  }

  public static void main(String[] args) {

    Scanner scan = new Scanner(System.in);
    int T = scan.nextInt();

    while (T > 0) {

      int N = scan.nextInt();
      factorial(N);
      T--;
    }

    scan.close();
  }

}

File: CCp/Arrays/TP/src/onestar/b/smallFact.ts
class Factorial {
  constructor() {}

  factorial = (n: number): void => {
    let fact = 1;
    for (let i = n; i >= 1; i--) {
      fact *= i;
    }
    console.log(fact);
  };
}

const factorial: Factorial = new Factorial();
factorial.factorial( 6 );
